workspacetools/add_arc2d.py

```diff
@@ -0,0 +1,22 @@
+from bpy.types import WorkSpaceTool
+
+from ..declarations import GizmoGroups, Operators, WorkSpaceTools
+from ..keymaps import disable_gizmos, tool_access, tool_select
+from ..stateful_operator.tool import GenericStateTool
+from ..stateful_operator.utilities.keymap import operator_access
+
+
+class VIEW3D_T_slvs_add_arc2d(GenericStateTool, WorkSpaceTool):
+    bl_space_type = "VIEW_3D"
+    bl_context_mode = "OBJECT"
+    bl_idname = WorkSpaceTools.AddArc2D
+    bl_label = "Add 2D Arc"
+    bl_operator = Operators.AddArc2D
+    bl_icon = "ops.gpencil.primitive_arc"
+    bl_widget = GizmoGroups.Preselection
+    bl_keymap = (
```

Comment:
These lines are repeated for every tool, maybe we could move it to keymaps.py and just import one keymap to avoid potential inconsistencies between tools.